components/chip/appearance.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Appearance
+page_title: Chip - Appearance
+description: Appearance settings of the Chip for Blazor.
+slug: chip-appearance
+tags: telerik,blazor,chip,appearance
+published: True
+position: 35
+---
+
+# Chip Settings
+
+You can control the appearance of the Chip by using the following attribute:
+
+* [FillMode](#fillmode)
+* [Size](#size)
+* [Rounded](#rounded)
+* [ThemeColor](#themecolor)
+
+
+## FillMode
+
+The `FillMode` controls how the Chip is filled. You can set it to a member of the `Telerik.Blazor.ThemeConstants.Chip.FillMode` class:
```

Comment:
It's good to explain non-self-evident parameters with different words, i.e. other than "filled". You can say something like "The FillMode affects the background and borders"